hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -114,8 +114,15 @@ public class ServicesState extends PartialNaryMerkleInternal
      */
     private MerkleScheduledTransactions migrationSchedules;
 
+    private final BootstrapProperties bootstrapProperties;
+
     public ServicesState() {
         // RuntimeConstructable
+        bootstrapProperties = null;
+    }
+
+    public ServicesState(final BootstrapProperties bootstrapProperties) {
```

Comment:
This is just used in tests ?

Reply:
Yes. This is so that I can inject an override for BootstrapProperties.

Reply:
May be add @VisibleForTesting annotation.